### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by TimoGlastra** [openwallet-foundation/askar#309]
We'd like to make a new release so we can get #307 in. However there have been changes to the native library as well which means we need to make a new release of Askar as well. I'm not sure if #291 introduced breaking changes? So do we need to go to the next 0.3.x, or to 0.4.x?

**Comment by swcurran** [comment]
@andrewwhitehead

**Comment by swcurran** [comment]
@TimoGlastra — can you or @berendsliedrecht weigh in on #312 and particularly, the NDK version that can be used for Android.  We need that PR merged before a new release — and we also want a new release.

Thanks

**Comment by ff137** [comment]
@TimoGlastra no breaking changes in #291 - just change to default behaviour that fetched records are now ordered by id. 

ACA-Py PR is also waiting for a new release: https://github.com/hyperledger/aries-cloudagent-python/pull/3173
We've merged that ordering functionality to our ACA-Py fork (with forked askar release), and everything's working well :-)

**Comment by ff137** [comment]
What's holding up the publishing of a new release? Something we can help with?

**Comment by swcurran** [comment]
Yes!!  The hold up is the android build of the react-native wrapper and a dependency mess, where some dependencies require a higher version of Rust, and other dependencies require a lower one (1.67 and 1.74).  I don’t really understand it all, but I know that the build is failing, and no one has had the time to get it resolved.  @cvarjao has a repo “aries-builder-images” that is related, but I don’t know the details.

My goal is to get that sorted out, and to get some documentation about the steps to do a release, so we are less dependent on individuals (although I’m not sure that is the issue here).  Documentation is good.

**Comment by cvarjao** [comment]
The main issue is that underline libraries such as askar, vdr, etc needs to align/agree on target android NDK (and minimum android version support). I am definitely not opposed with keeping the NDKs up-to-date. 
Google Play services v23.30.99+ (August 2023) [drops support](https://android-developers.googleblog.com/2023/07/google-play-services-discontinuing-updates-for-kitkat.html) for API levels below 21, so going to ndk 25+ (28 is the most recent), I think should be fine

**Comment by andrewwhitehead** [comment]
Rust 1.68 moved to NDK 25 as the minimum supported version and it looks like that is the minimum for Bifold at the moment, so I don't think we need to increase it further (for now).

**Comment by TimoGlastra** [comment]
I discussed this with @berendsliedrecht and he said the approach should just be to try it with newer NDK versions and make sure all the SDKs still work

**Comment by ff137** [comment]
@berendsliedrecht note that Publish to PyPI errored: [github action link](https://github.com/openwallet-foundation/askar/actions/runs/13029197376/job/36344892560)

This needs a version bump: https://github.com/openwallet-foundation/askar/blob/main/wrappers/python/aries_askar/version.py
